packages/@react-types/dialog/src/index.d.ts

```diff
@@ -36,19 +36,16 @@ export interface SpectrumBaseDialogProps extends HTMLAttributes<HTMLElement> {
 }
 
 export interface SpectrumDialogProps extends DOMProps, StyleProps {
-  /** The contents as semantic elements to display in the dialog. */
+  /** The contents of the Dialog. */
   children: ReactNode,
   /**
-   * [Slots](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/slot)
-   * defining semantic names for dialog children.
+   * Object containing the props that each Dialog child receives,
+   * see [slots](./Slots.html).
    */
   slots?: Slots,
-  /** How wide the dialog should be or make fullscreen. */
+  /** How size of the Dialog will be determined. */
```

Comment:
Is probably worth mentioning that this only applies to the modal Dialog I believe. Maybe something like "The size of the Dialog. Only applies to "modal" type Dialogs.